benchmarks/benchmarks/bench_random.py

```diff
@@ -34,11 +34,11 @@ class Randint(Benchmark):
 
     def time_randint_fast(self):
         """Compare to uint32 below"""
-        np.random.randint(0, 2**30, size=10**5)
+        np.random.randint(low=0, high=2**30, size=10**5)
```

Comment:
changing non-test code is imo ok, but when changing test code you need to make sure that no coverage of the old still supported way of calling it is lost. It is not obvious if that has been done.
Please add new test cases for the old way of calling the function if you have removed them.

I may have overlooked it but I also see now tests for the new `None` cases